articles/node-vs-nuxt/index.md

```diff
@@ -0,0 +1,160 @@
+| Built on            | Chrome V8 Engine                                                    | Vue.js                                                                       |
+| Languages Supported | JavaScript, TypeScript                                              | JavaScript, TypeScript                                                       |
+| Open-Sourced        | Yes                                                                 | Yes                                                                          |
+
+### Create a Node.js App
+
+Let us create a basic Node.js application.
+
+- To start, create a directory called hello-node.
+
+```
+mkdir hello-node
+
+cd hello-node
+```
+
+- Now, let us initiate a Node.js application by using the Node Package Manager.
+
+```
+npm init
+```
+
+![Node App Creation](/engineering-education/node-vs-nuxt/hello-node.png)
+
+- The above command initializes a project and creates a `package.json` file in the current directory. The package.json file is used to manage dependencies of the application.
+
+- Create a new file `index.js`.
+
+- For our script, let us install dependencies using the node package manager.
+
+```
+npm i emoji-random node-emoji
+```
+
+- Now, let us start writing our Node.js script that prints a random emoji to the console.
+
+```js
+// Importing process.stdin to take user input
+let std_input = process.stdin;
+
+// Importing packages
+let re = require("emoji-random");
+let emoji = require("node-emoji");
+
+console.log("1. print");
+console.log("2. exit");
+
+// When data is entered into stdin, this block executes
+std_input.on("data", (data) => {
+  if (data != "exit") {
+    let randEmoji = re.random();
+    var finalEmoji = emoji.get(randEmoji);
+
+    // Print to the console
+    console.log(
+      "Here is the " + randEmoji.split(":")[1] + "emoji: " + finalEmoji
+    );
+    process.exit();
+  } else {
+    console.log(data);
+    console.log("exiting!");
+    process.exit();
+  }
+});
+```
+
+- We have thus created a Node.js application.
+
+### Create a Nuxt App
+
+- To create a Nuxt application, we use the `create-nuxt-app` utility.
+
+```
+npx create-nuxt-app hello-nuxt
+```
+
+![Create Nuxt App](/engineering-education/node-vs-nuxt/create-nuxt-app.png)
+
+- Nuxt creates a folder structure for us. It also takes care of routing. Every `.vue` file under the `pages` folder is a route.
+
+- Let us run the application.
+
+```
+npm run dev
+```
+
+- The application is initialized and runs on the URL, `localhost:3000`.
+
+![Run Nuxt App](/engineering-education/node-vs-nuxt/npm-run-dev.png)
+
+- The web application can be viewed on the browser.
+
+![Web Application](/engineering-education/node-vs-nuxt/nuxt-app.png)
+
+- To modify our web app, we can start editing the `pages/index.vue` file.
+
+- We have thus created a Nuxt application.
```

Comment:
You should use provide screenshots of the finished results. Perhaps include code examples of fully-fledged Node.js and Nuxt web applications so readers can fully appreciate the difference between them. Also, you could highlight major sites that are powered by Node.js and Nuxt.